### ISSUE TITLE
Account > Overview : History filterable into (last/weekly/montly) 

### ISSUE THREAD

**Issue by c055** [cryptonomex/graphene-ui#498]
<img width="984" alt="Filter able history" src="https://cloud.githubusercontent.com/assets/6280810/11218600/284c9026-8d56-11e5-9347-57a1d2aa4f3c.png">

just brainstorming .. Feedback appreciated

**Comment by xeroc** [comment]
Other possible filters:
- by asset
- by receiver
- memo (search)
- date

**Comment by c055** [comment]
yep nice - what i'm really loving on the Darkwallet GUI is that all balances are summed up to a weekly / monthly report ..

**Comment by c055** [comment]
also referenced to https://github.com/cryptonomex/graphene-ui/issues/491 ... Ability to download history… for accounting etc. purposes

**Comment by c055** [comment]
also .. https://github.com/cryptonomex/graphene-ui/issues/494

**Comment by wmbutler** [comment]
I still haven't had a change to touch this. Will move to 1216
